swabbie-aws/src/main/kotlin/com/netflix/spinnaker/swabbie/aws/edda/Edda.kt

```diff
@@ -175,6 +176,23 @@ class Edda(
     }
   }
 
+  // TODO: This should be updated as it would fail once edda fixes this endpoint
+  // see pattern in other functions
+  // See note in EddaService.getLaunchTemplateVersions
+  override fun getLaunchTemplateVersions(params: Parameters): List<AmazonLaunchTemplateVersion> {
+    val result = call(params, SERVER_GROUP) {
+      eddaService: EddaService ->
+      eddaService.getLaunchTemplateVersions()
+    } ?: emptyList()
+
+    return result.flatMap { it.versions }
+  }
+
+  data class EddaLaunchTemplaVersion(
```

Comment:
This is to workaround the current edda payload. If they remove this endpoint in favor of the more traditional endpoints, this will begin to fail. left a TODO in there for when that happens :)